plugin.video.zdf_de_2016/addon.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.zdf_de_2016" name="ZDF Mediathek 2016" version="1.0.2" provider-name="Generia">
+<addon id="plugin.video.zdf_de_2016" name="ZDF Mediathek 2016" version="1.0.3" provider-name="Generia">
   <requires>
-    <import addon="xbmc.python" version="2.24.0"/>
+    <import addon="xbmc.python" version="2.14.0"/>
```

Comment:
I see why this can make sense, but I think our preferred way is to keep the dependency version in sync with repo branch version.
@MartijnKaijser correct me when I´m wrong.